circt/README.md

```diff
@@ -0,0 +1,15 @@
+# CIRCT Utilities Container
```

Comment:
Docker run commands are soooo boilerplate and cumbersome. You may want to provide some scripts like this example. For instance, start the docker container, run a particular command, and ["do the right thing"](https://github.com/llvm/circt/blob/main/utils/run-docker.sh).

Reply:
Will do. I'll probably add one each for `firtool`, `circt-opt`, and `circt-translate`.